packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dropDownEditor.tests.js

```diff
@@ -2127,7 +2127,7 @@ QUnit.module('aria accessibility', () => {
         assert.equal($input.attr('aria-expanded'), 'false', 'aria-expanded property on closed');
     });
 
-    QUnit.test('component with fieldTemplate should retain aria attributes after interaction (T1230696, T1230971)', function(assert) {
+    QUnit.test('component with fieldTemplate should have proper aria attributes after interaction (T1230696, T1230971, T1230706)', function(assert) {
```

Comment:
lets create a separate test for this or refactor. as far as i can see it's possible to use tests parameters to provide aria-invalid and aria-autocomplete as params